k8s/perftest/common/probate/probate-caveats-fe.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: probate-caveats-fe
+  namespace: probate
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.nodejs: glob:prod-*
+spec:
+  releaseName: probate-caveats-fe
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: probate-caveats-fe
+    version: 1.1.13
+  values:
+    nodejs:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      environment:
+        EQUALITY_URL: https://pcq.perftest.platform.hmcts.net
+      image: hmctspublic.azurecr.io/probate/caveats-fe:prod-f27f4b64
```

Comment:
Same as few above comments, may be we should leave it to flux to pull the latest image by itself-
image: hmctspublic.azurecr.io/probate/caveats-fe:latest